### PR TITLE
Changing the template

### PR DESCRIPTION
**Pull request by shijatsu**
to the PM mockup

### CODE REVIEW

File: Workbooks/Storage/Overview/Overview.workbook [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "version": "KqlParameterItem/1.0",
         "query": "",
         "crossComponentResources": [
-          "value::selected"
+          "{Subscription}"
```
Comment (gardnerjr) [inline Workbooks/Storage/Overview/Overview.workbook]: I don't think this part is right?  this says this very first parameter depends on another parameter called {subscription} that doesn't exist yet?

File: Workbooks/Storage/Overview/Overview.workbook [hunk 2]
```diff
@@ -44,16 +44,19 @@
...
             "typeSettings": {
-              "limitSelectTo": 10,
+              "limitSelectTo": 25,
               "additionalResourceOptions": []
             },
             "timeContextFromParameter": null,
             "queryType": 1,
-            "resourceType": "microsoft.resourcegraph/resources"
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": [
```
Comment (gardnerjr) [inline Workbooks/Storage/Overview/Overview.workbook]: this actually has a selected subscription inside, should be value::selected or all or whatever, no specific resources should be selected

File: Workbooks/Storage/Overview/Overview.workbook [hunk 3]
```diff
@@ -174,19 +177,82 @@
...
+          },
+          "showBorder": false
+        }
+      },
+      "conditionalVisibility": null,
+      "customWidth": "20",
+      "name": "query - 20"
+    },
+    {
+      "type": 3,
```
Comment (gardnerjr) [inline Workbooks/Storage/Overview/Overview.workbook]: if this health stuff isn't ready, just remove it?  (right now it's still running)

File: Workbooks/Storage/Overview/Overview.workbook (hunk 1 above)
Comment (shijatsu) [reply Workbooks/Storage/Overview/Overview.workbook]: Fixed in new PR

File: Workbooks/Storage/Overview/Overview.workbook (hunk 2 above)
Comment (shijatsu) [reply Workbooks/Storage/Overview/Overview.workbook]: Fixed in new PR

File: Workbooks/Storage/Overview/Overview.workbook (hunk 3 above)
Comment (shijatsu) [reply Workbooks/Storage/Overview/Overview.workbook]: fixed in new PR

**Comment by shijatsu** [comment]
Opened a new PR

**Pull request closed without merging**